js/error-message.js

```diff
@@ -0,0 +1,21 @@
+const EMAIL_REQUIRED = '이메일을 입력해 주세요.';
+const EMAIL_INVALID ='올바른 이메일 주소가 아닙니다.'
+const CHECK_EMAIL ='이메일을 확인해 주세요.'
+const EMAIL_ALREADY_USE ='이미 사용 중인 이메일 입니다.'
+const PW_REQUIRED ='비밀번호를 입력해 주세요.'
+const CHECK_PW ='비밀번호를 확인해 주세요.'
+const PW_INVALID ='비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.'
+const PW_NOT_MATCH ='비밀번호가 일치하지 않아요.'
+
+const errorMessage = {
```

Comment:
에러 메시지 상수들의 모음이므로 해당 객체도 대문자로 작성하면 좋겠네요.

```suggestion
const ERROR_MESSAGE = {
```